product_links/__openerp__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     'name': 'Product Links',
-    'version': '7.0.0',
+    'version': '8.0.0',
```

Comment:
This version is incorrect. It should be 1.0. Odoo will add 8.0 itself at the beginning.